client/public/clientJS/passwordReset.js

```diff
@@ -21,7 +21,9 @@ $(document).ready(function () {
             },
             error: function(response, textStatus, errorThrown) {
                 $("#submit-btn").prop("disabled",false);
-                showToast(response.responseJSON.err);
+                let txtToJson = response.responseJSON;
+                let { msg } = txtToJson.error;
+                showToast(msg);
```

Comment:
you can probably just do response.responseJSON.error.msg?

Reply:
Yea, this looked bit more nice xD